vmdb/app/helpers/ems_container_helper/textual_summary.rb

```diff
@@ -43,10 +42,6 @@ def textual_hostname
     {:label => "Hostname", :value => @ems.hostname}
   end
 
-  def textual_ipaddress
-    {:label => "IP Address", :value => @ems.ipaddress}
```

Comment:
@simon3z I've noticed 'hostname' row always contains hostname/ip and ip is always empty (see screen shots) so I decided to drop altogether.

@abonas I could test, however I do not have any non container providers currently.
I can add ovirt as a cloud provider I assume? 
I could check that tomorrow.